examples/md/md_without_nf.py

```diff
@@ -0,0 +1,253 @@
+    # m.fs.nf = NanofiltrationZO(property_package=m.fs.properties)
+    # m.fs.P1 = Pump(property_package=m.fs.properties)
+
+    # m.fs.P1.efficiency_pump.fix(0.80)  # pump efficiency [-]
+    # m.fs.P1.outlet.pressure[0].fix(10e5)
+
+    # # fully specify system
+    # m.fs.nf.properties_permeate[0].pressure.fix(101325)
+    # m.fs.nf.recovery_vol_phase.fix(0.5)
+    # m.fs.nf.flux_vol_solvent.fix(1.446759259259259e-5)
+    # m.fs.nf.area.fix(499.44685)
+    # Create QGESS costing block
+    m.fs.costing = QGESSCosting()
+    
+    # Apply costing with detailed parameters
+    m.fs.costing.build_process_costs(
+        # Capital cost factors
+        cost_factor=1.58, 
+        piping_materials_and_labor_percentage=20,
+        electrical_materials_and_labor_percentage=20,
+        instrumentation_percentage=8,
+        plants_services_percentage=10,
+        process_buildings_percentage=40,
+        auxiliary_buildings_percentage=15,
+        site_improvements_percentage=10,
+        equipment_installation_percentage=17,
+        field_expenses_percentage=12,
+        project_management_and_construction_percentage=30,
+        process_contingency_percentage=15,
+        
+        # Labor cost parameters
+        labor_types=[
+            "skilled",
+            "unskilled",
+            "supervisor",
+            "maintenance",
+            "technician",
+            "engineer",
+        ],
+        labor_rate=[26.08, 19.08, 30.39, 22.73, 21.97, 45.85],  # USD/hr
+        labor_burden=25,  # % fringe benefits
+        operators_per_shift=[2, 0, 0, 0, 0, 0],
+        hours_per_shift=8,
+        shifts_per_day=3,
+        operating_days_per_year=365,
+        
+        # Product and efficiency parameters
+        mixed_product_sale_price_realization_factor=0.65,
+        efficiency=0.80,  # power usage efficiency
+        resources=[],
+        rates=[],
+        # O&M costs
+        fixed_OM=True,
+        variable_OM=True,
+        land_cost=1,
+        CE_index_year="UKy_2019",
+        
+        # Units to include in costing
+        watertap_blocks=[
+            m.fs.MD, 
+            # m.fs.nf, 
+            # m.fs.P1, 
+            m.fs.hx, 
+            m.fs.heater,
+            m.fs.mixer, 
+            m.fs.pump_feed, 
+            m.fs.pump_brine, 
+            m.fs.pump_permeate
+        ]
+    )
+    
+    # Initialize costing
+    QGESSCostingData.costing_initialization(m.fs.costing)
+    QGESSCostingData.initialize_fixed_OM_costs(m.fs.costing)
+    
+    # Calculate LCOW (Levelized Cost of Water)
+    denominator = pyunits.convert(
+        m.fs.permeate.properties[0].flow_vol, 
+        to_units=pyunits.m**3 / pyunits.year
+    )
+    m.fs.costing.prommis_LCOW = Expression(
+        expr=m.fs.costing.annualized_cost / denominator * 1e6
+    )
+    
+    # Set objective
+    m.fs.objective = Objective(expr=m.fs.costing.prommis_LCOW)
+    
+
+    
+    
+    
+    
+    
+    recovery_range= 0.5, #np.linspace(0.05,.95,100)
```

Comment:
solving for 50% MF system recovery. can uncomment the np.linspace bit to cover a range of recoveries